examples/list-setters-simple/README.md

```diff
@@ -0,0 +1,36 @@
+# list-setters: Simple Example
+
+### Overview
+
+In this example, we will see how to list setters in a package.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/list-setters-simple
+```
+
+### Function invocation
+
+Invoke the function by running the following command:
+
+```shell
+$ kpt fn eval --image gcr.io/kpt-fn/list-setters:unstable
+```
+
+### Expected result
+
+```shell
+[RUNNING] "gcr.io/kpt-fn/list-setters:unstable"
+[PASS] "gcr.io/kpt-fn/list-setters:unstable"
+  Results:
+    [INFO] Name: env, Value: [stage, dev], Type: array, Count: 1
+    [INFO] Name: nginx-replicas, Value: 3, Type: str, Count: 1
```

Comment:
int

Reply:
I feel full names would be better for display purposes and will be consistent with legacy setters. WDTY?

```
string
integer
boolean
float
```

Reply:
as discussed offline keeping as is for now